### ISSUE TITLE
Separate general, rebrand CSS from student CSS

### ISSUE THREAD

**Issue by shaun-technovation** [Iridescent-CM/technovation-app#4421]
Right now some of the rebranding styles are contained in student-specific CSS files, we should separate the generic styles to a generic CSS file.

There might be other spots, but here's at least one of the files w/ general rebranding CSS:
`app/javascript/stylesheets/student_profile.css`

**Comment by viviancan** [comment]
We may need to review `tabs.css`. The `.tw-active` class is causing some weirdness with the ChA tabs on the dashboard side nav

**Comment by shaun-technovation** [comment]
I separated out some navigation, button, and layout styles from `student_profile.css`.

I had originally thought that having these generic styles in `student_profile.css` was why we needed to include student styles on the new chapter ambassador dashboard. But that's not the case, in a nutshell, the old chapter ambassador styles (for the old dashboard aka Chapter Admin Activity) were interfering with the new chapter ambassador dashboard; the student styles don't include these styles which is why it fixed the issue. Anywho, I created new a "chapter ambassador rebrand" CSS file to address this issue (instead of including the student styles) that includes the relevant styles for the new dashboard.
